docs/toc.md

```diff
@@ -60,6 +60,7 @@
 ## [StaggeredPanel](controls/StaggeredPanel.md)
 ## [TextToolbar](controls/TextToolbar.md)
 ## [TileControl](controls/TileControl.md)
+## [WebView](controls/WebView.md)
```

Comment:
Should we add this to a new category called Win32 controls?

Reply:
Or simply specify in the name that this is a win32 control ex: `WebvView (win32)`